README.md

```diff
@@ -30,13 +31,13 @@ Follow the steps below to build and view the site locally.
 
 ## Viewing the output
 
-You can use [npm] with [http-server] to view the output:
+  Run task `serve` to install [http-server] and view the output:
 
-```shell
-npx http-server build/html -c-1
-```
+  ```shell
+  task serve
+  ```
```

Comment:
Oops, I didn't realize this wasn't a bullet point in the first place. These should all not be indented.